code/game/objects/items/ego_weapons/non_abnormality/shi.dm

```diff
@@ -0,0 +1,65 @@
+//Shi has 2 different modes. Dash Attacks and Boundary of Death.
+//Shi Assassin (Current one being used right now) uses Boundary of death.
+
+
+//Boundary of death uses
+//Grade 5
+/obj/item/ego_weapon/city/shi_association
+	name = "shi association sheathed blade"
+	desc = "A blade that is standard among blade lineage."
+	special = "Use this weapon in hand to immobilize yourself for 1 second, cut your HP by 10%, and deal 2x damage in pale."
+	icon_state = "shi_association"
+	force = 46
```

Comment:
```suggestion
	force = 44
```
Symbolistic change fitting their style. This or
```suggestion
	force = 50
```
B/c it's the BoD roll. Either or honestly.